luthier/src/test/groovy/com/yahoo/bard/webservice/config/luthier/FactoryParkTest.groovy

```diff
@@ -0,0 +1,14 @@
+package com.yahoo.bard.webservice.config.luthier
+
+import spock.lang.Specification
+
+class FactoryParkTest extends Specification {
+    void setup() {
+    }
+
+    void cleanup() {
+    }
```

Comment:
Neither one of these methods need to be defined unless you plan on using them.

Reply:
Removed.